aten/src/ATen/Quantizer.cpp

```diff
@@ -0,0 +1,134 @@
+#include <ATen/ATen.h>
+#include <ATen/Quantizer.h>
+#include <ATen/QTensorImpl.h>
+#include <ATen/NativeFunctions.h>
+#include <ATen/native/TensorFactories.h>
+
+namespace at {
+
+std::shared_ptr<Quantizer> make_per_layer_affine_quantizer(double scale, int64_t zero_point) {
+  return std::make_shared<PerLayerAffineQuantizer>(static_cast<float>(scale), static_cast<int32_t>(zero_point));
```

Comment:
It seems a little weird that we downcast first, and then pass it to the constructor

Reply:
Let us enforce float for scale and uint8 for zero_point, since we support only 8 bit quantization.

Reply:
@ezyang how can we do this differently?

Reply:
@raghuramank100 sure